provisioning-app/ocp-config/secret.yml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Template
+objects:
+- apiVersion: v1
```

Comment:
Is this secret being used at all? I suspect it was only there to help build the image (to clone the repo).